spec-0004/index.md

```diff
@@ -18,38 +19,99 @@ If relevant, include examples of how the new functionality would be used,
+into a release, at which point it becomes much harder to change or revert
+something.
 
-1. Testing against nightly wheels will help identify when changes to the main branch
-   of the widely-used projects break downstream projects.
-2. By using the nightly wheels projects will not have to build packages from source,
-   which can be difficult and time consumming.
+Regularly creating nightly wheels for your project allows projects that depend on
+you to give feedback about upcoming changes. As with testing against nightlies of
+your dependencies this gives your dependents a chance to report problems before they
+find their way into a release.
 
 ## Implementation
 
-<!--
-Discuss how this would be implemented.
--->
+This section outlines how to implement using and building nightly wheels. We assume your
+project already has some amount of CI infrastructure and that you will have to fit this
+in with the existing setup. In the notes section we link to projects who have implemented
+this in their setup to give you examples of complete setups.
+
+### Test with Nightly Wheels
+
+We recommend that projects add a weekly cron job to run their tests using nightly versions
+of their dependencies. The cron job should automatically open an issue on your repository
+when it encounters an error.
+
+If you spot a problem please investigate if this is due to a known deprecation or
+bug fix. If you think it is neither, please report it to the relevant upstream project.
+
+To install the nightly version of your dependencies check which of them are available
+at https://anaconda.org/scientific-python-nightly-wheels/. For example to install the NumPy, pandas and scipy nightlies use:
+
+```
+pip install --pre --upgrade --extra-index https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy pandas scipy
+```
+
+Complete examples of how projects implement this in their CI setup are linked in the Notes section.
 
-We recommend that projects add a cron job to test against the nightly wheels and
-when errors are encountered to automatically open an issue in their project so they
-can investigate.
+### Build Nightly Wheels
 
-The nightly wheels can be found [here](https://anaconda.org/scientific-python-nightly-wheels/).
+There are a few steps to implementing this for your project:
 
-We have a GitHub action to upload new nightly wheels and remove old ones:
-https://github.com/scientific-python/upload-nightly-action
+1. Get access to https://anaconda.org/scientific-python-nightly-wheels/, the location used to host nightly wheels
+2. Setup a CI step that builds wheels for your project
+3. Setup a CI step that uploads wheels to https://anaconda.org/scientific-python-nightly-wheels/
 
-### Core Project Endorsement
+For step (1) contact nightly-wheels@scientific-python.org. Deposit the token you get as a [secret on your repository](https://docs.github.com/en/actions/security-guides/encrypted-secrets) named `UPLOAD_TOKEN`.
```

Comment:
Who will be behind this email ?

Reply:
Good question. I'd say all those who currently have admin access to the new nightlies area? Plus those who want to volunteer to work on this?

Reply:
Rather than email, what about initiating the discussion in a github issue?

Reply:
> Rather than email, what about initiating the discussion in a github issue?

See https://github.com/scientific-python/specs/pull/195#discussion_r1204850632 - I think its a good idea

Reply:
I'll mark this as resolved as the SPEC is now proposing to use a GitHub repo instead of the email. The question of who will feel responsible for responding to those issues is still open though.